### ISSUE TITLE
Update Fast Classpath Scanner dependency

### ISSUE THREAD

**Issue by hollingsworthd** [MachinePublishers/jBrowserDriver#129]
See #41

**Comment by ArloL** [comment]
fast-classpath-scanner 1.9.19 was released and now my CI builds fail with `java.lang.NoClassDefFoundError: io/github/lukehutch/fastclasspathscanner/scanner/ClasspathFinder` since they changed the package of `ClasspathFinder` from `io.github.lukehutch.fastclasspathscanner.scanner` to `io.github.lukehutch.fastclasspathscanner.classpath`.

My workaround for now is to fix the version of fast-classpath-scanner:

~~~
<dependencyManagement>
    <dependencies>
        <dependency>
            <groupId>io.github.lukehutch</groupId>
            <artifactId>fast-classpath-scanner</artifactId>
            <version>1.9.18</version>
        </dependency>
    </dependencies>
</dependencyManagement>
~~~

What do you think about fixed versions in the pom instead of version ranges? That would prevent things like this failing builds.

**Comment by hollingsworthd** [comment]
Thanks for reporting this! jBrowserDriver update released to fix this: v0.14.5

**Comment by hollingsworthd** [comment]
Good point, I will probably switch to fixed versions

**Comment by nycoder67** [comment]
This got me too. Fixed versions has my vote.

**Comment by hollingsworthd** [comment]
Version are fixed as of 0.14.6
